src-tauri/Cargo.toml

```diff
@@ -18,7 +18,7 @@ tauri-build = { version = "1.5.1", features = [] }
 tauri = { version = "1.6.0", features = [ "window-close", "window-set-title", "path-all", "dialog", "dialog-confirm", "dialog-save", "dialog-open", "dialog-ask", "fs-all", "shell-open", "devtools"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-trajoptlib = { git = "https://github.com/SleipnirGroup/TrajoptLib.git", rev = "325ddaa659cd36db8a87da363b66f66837df3e8f", features = ["sleipnir"] }
+trajoptlib = { git = "https://github.com/bruingineer/TrajoptLib.git", rev = "b3ae9e4bde9c67436e005f7c0a5547d203fca1da", features = ["sleipnir"] }
```

Comment:
This will need to be updated after merge of https://github.com/SleipnirGroup/TrajoptLib/pull/140